platform/podjs/src/browserPod.ts

```diff
@@ -16,7 +16,7 @@ import * as zip from "@zip.js/zip.js";
 import endpointsJson from "../../../../polyPod-config/endpoints.json";
 import { Manifest, readManifest } from "./manifest";
 
-const DB_NAME = "polypod";
+const DB_PREFIX = "polypod";
```

Comment:
Nit: You hard code the `:` below, I think it'd be cleaner if this was included in the prefix, i.e. `polypod:` instead of `polypod` here.

Reply:
Done. Though, I might argue that the colon is part of the structure, ant therefore it makes sense to have it in the template string, but then this variable should probably be called `DB_NAMESPACE` (not *prefix*). But I don't think it matters too much.